modeltranslation/manager.py

```diff
@@ -69,57 +69,17 @@ def append_fallback(model, fields):
     If translated field is encountered, add also all its fallback fields.
     Returns tuple: (set_of_new_fields_to_use, set_of_translated_field_names)
     """
-    from django.db.models.constants import LOOKUP_SEP
```

Comment:
do not remove this part of code because it is one of things why we have this fork